### PR TITLE
Update create-package template to better handle converged packages

### PR DESCRIPTION
**Pull request by ecraig12345**
#### Description of changes

If the package being created is converged (based on a user answer), run `migrate-converged-pkg` automatically during creation.

Also remove the parts about react-examples.

#### Focus areas to test

Created a package locally. It worked.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `57.532 kB`<br />`18.112 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `54.953 kB`<br />`15.667 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `23.182 kB`<br />`6.986 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `25.642 kB`<br />`7.682 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenseBadge</abbr>                                                                       |          `237 B`<br />`177 B` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `25.501 kB`<br />`7.738 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |   `30.758 kB`<br />`8.689 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |   `27.526 kB`<br />`8.413 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |   `33.637 kB`<br />`9.597 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |    `34.727 kB`<br />`8.39 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |   `13.471 kB`<br />`5.068 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |    `9.317 kB`<br />`3.925 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |    `8.468 kB`<br />`3.557 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |     `9.695 kB`<br />`3.99 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `8.783 kB`<br />`3.741 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `169.173 kB`<br />`48.002 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.188 kB`<br />`10.658 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |   `15.342 kB`<br />`5.585 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |    `9.771 kB`<br />`3.975 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |  `31.319 kB`<br />`11.305 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |    `8.952 kB`<br />`3.708 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `11.646 kB`<br />`4.699 kB` |
| <samp>react-make-styles</samp> <br /> <abbr title='bundle-size/makeStaticStyles-runtime.fixture.js'>makeStaticStyles (runtime)</abbr>                                         |     `7.59 kB`<br />`3.321 kB` |
| <samp>react-make-styles</samp> <br /> <abbr title='bundle-size/makeStyles-runtime.fixture.js'>makeStyles + mergeClasses (runtime)</abbr>                                      |   `22.175 kB`<br />`8.366 kB` |
| <samp>react-make-styles</samp> <br /> <abbr title='bundle-size/makeStyles.fixture.js'>makeStyles + mergeClasses (build time)</abbr>                                           |    `2.558 kB`<br />`1.204 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `105.789 kB`<br />`32.202 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `108.065 kB`<br />`32.575 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 |  `101.153 kB`<br />`30.37 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |    `6.725 kB`<br />`2.237 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePopper</abbr>                                                                         |   `23.145 kB`<br />`7.942 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `15.147 kB`<br />`5.573 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/RangedSlider.fixture.js'>RangedSlider</abbr>                                                                        |    `41.41 kB`<br />`11.97 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |  `34.788 kB`<br />`10.855 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |   `26.606 kB`<br />`8.557 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `11.338 kB`<br />`4.418 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |   `14.407 kB`<br />`4.724 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                                  |    `25.712 kB`<br />`6.17 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                                |   `17.215 kB`<br />`5.116 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `45.543 kB`<br />`15.541 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                                                                       |          `213 B`<br />`170 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/8df3ce7b4d72bc2a155b93a6a35c13d624432f7c'>8df3ce7b4d72bc2a155b93a6a35c13d624432f7c</a></sub>
<div id="bundleSizeComment9423"/>

File: scripts/create-package/plopfile.ts [hunk 1]
```diff
@@ -56,41 +57,34 @@ module.exports = (plop: NodePlopAPI) => {
...
 
       const destination = `packages/${packageName}`;
-      const exampleRoot = `packages/react-examples`;
       const globOptions: AddManyActionConfig['globOptions'] = { dot: true };
 
       // Get derived template parameters
       const data = {
         packageNpmName: '@fluentui/' + packageName,
-        packageVersion: '9.0.0-alpha.0',
+        packageVersion: answers.isConverged ? '9.0.0-alpha.0' : '0.1.0',
```
Comment (ling1726) [inline scripts/create-package/plopfile.ts:81]: we're in beta now :)

```suggestion
        packageVersion: answers.isConverged ? '9.0.0-beta.0' : '0.1.0',
```

File: scripts/create-package/plopfile.ts [hunk 2]
```diff
@@ -56,41 +57,34 @@ module.exports = (plop: NodePlopAPI) => {
...
       },
       {
         type: 'confirm',
         name: 'publish',
-        message: answers =>
-          answers.hasExamples
-            ? 'Should the package be published right away (must be Yes for packages with example scaffolding)?'
-            : 'Should the package be published right away?',
-        default: (answers: Answers) => !!answers.hasExamples,
+        message: 'Should the package be published right away?',
```
Comment (ling1726) [inline scripts/create-package/plopfile.ts:67]: perhaps let's just remove this and always set `private` at the worst case you'll realize the next day you didn't release what you wanted to and just rerelease

File: scripts/create-package/plopfile.ts [hunk 3]
```diff
@@ -204,20 +190,24 @@ function replaceVersionsFromReference(
...
-        );
+        // Record the error and wait to throw until later for better logs
+        errorPackages.push(`${depPkg} (${depType})`);
       }
     }
   }
-  return hasError;
+  if (errorPackages.length) {
+    throw new Error(
+      `Couldn't determine appropriate version of ${errorPackages.length} packages from ${referencePackages.join(
```
Comment (ling1726) [inline scripts/create-package/plopfile.ts:200]: perhaps we could stop doing this and set references packages with `*` syncpack should scream on this and prompt the dev to actually set the correct versions

**Pull request merged**